pom.xml

```diff
@@ -50,6 +50,11 @@
             <artifactId>kafka-schema-registry-maven-plugin</artifactId>
             <version>6.0.1</version>
         </dependency>
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-avro-serializer</artifactId>
+            <version>6.0.1</version>
+        </dependency>
```

Comment:
If the package is used just for testing, better to scope it as "test" and move under the test plugins section of the pom file

Reply:
Good call for scope. Only reason I was kept it next to other confluent dependencies because they are kind of expected to work on same version, I noticed some weird issues when serializer was as same as other confluent dependencies.